Firebase/Messaging/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 2015-06-12 -- v3.0.2
+- Add support for global data collection switch in FIRApp.
+- Adding warning message when topic format is not right.
```

Comment:
Consider rephrasing to `Added a warning message when subscribing to topics with incorrect name formats.`

Reply:
Done